.github/workflows/build_packages.yml

```diff
@@ -36,9 +38,14 @@ jobs:
 
       - name: Build iree-turbine release candidate
```

Comment:
> We also can't make multiple different versions for the same day.

Yep. I thought about making a change like this one but figured that nightly releases are "best effort" anyways, and overwriting a nightly release if we want to pick up some commits before the next day isn't a major issue. I'm not opposed to adding a number suffix and allowing multiple releases per day, if you put the work in to make that possible. It's just more code to write and maintain.

Reply:
I wanted this to improve reproducibility as we are using pinned versions for the projects that depend on IREE Turbine.